frontend/src/models/api/AcquisitionFile.ts

```diff
@@ -29,3 +30,9 @@ export interface Api_AcquisitionFileProperty extends Api_ConcurrentVersion, Api_
   property?: Api_Property;
   acquisitionFile?: Api_AcquisitionFile;
 }
+
+export interface Api_AcquisitionFilePerson extends Api_ConcurrentVersion, Api_AuditFields {
+  id?: number;
+  personId: number;
+  personProfileTypeCode: string;
```

Comment:
Given that the API does not guarantee non-null values, please change the types to be possibly undefined

Reply:
done